app/views/buses/arrivals.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Buses</h1>
+
+<% @arrivals.each do |stop| %>
+  <ul>
+    <% stop.each do |bus| %>
+      <li>
+        <%= bus['lineName'] %>
+        to <%= bus['destinationName'] %>
+        arriving in <%= bus['timeToStation'] / 60 %> minutes
```

Comment:
Might be nicer to say "due" here instead of 0 minutes?

Also just to double check the division will floor the number in the end and you don't get weird decimal times?

Reply:
I believe the output of division in ruby takes the type of the divider, or `b` in `a / b`, so this is essentially integer division which floors